lib/src/component_common.dart

```diff
@@ -66,7 +66,8 @@ abstract class FluxComponentCommon<ActionsT, StoresT> extends react.Component
             value: (_) => (_) => redraw())
           ..addAll(getStoreHandlers());
     handlers.forEach((store, handler) {
-      manageStreamSubscription(store.listen(handler));
+      var subscription = store.listen(handler);
+      getManagedDisposer(() async => subscription.cancel());
```

Comment:
I think there's a `.stream` getter on the store or something. But it's not a big deal.